support/Environments/ocean_gcc.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Environment for the ocean cluster, located at Cal State Fullerton.
+#
+# Access ocean via `ssh ocean.fullerton.edu`
+#
+# For questions regarding ocean or to request access, please contact
+# Geoffrey Lovelace by email (glovelace at fullerton dot edu).
+# Access is normally restricted to members of Cal State Fullerton's
+# Gravitational-Wave Physics and Astronomy Center and their collaborators.
+
+spectre_setup_modules() {
+    echo "All modules on ocean are provided by the system"
+}
+
+spectre_unload_modules() {
```

Comment:
We've generally been doing 4 space indentations inside these functions. Would you mind doing that? Thanks :)

Reply:
It's not documented, I'm planning a PR with more code review things that I'll add it too. All the current env files do that, though.

Alright, sounds good :)